docs/refs.bib

```diff
@@ -285,4 +317,127 @@ @inproceedings{barocas2017problem
   author={Barocas, Solon and Crawford, Kate and Shapiro, Aaron and Wallach, Hanna},
   booktitle={9th Annual conference of the special interest group for computing, information and society},
   year={2017}
+}
+
+@inproceedings{shahhosseini2020optimizing,
+  title={Optimizing ensemble weights for machine learning models: A case study for housing price prediction},
+  author={Shahhosseini, Mohsen and Hu, Guiping and Pham, Hieu},
+  booktitle={Smart Service Systems, Operations Management, and Analytics: Proceedings of the 2019 INFORMS International Conference on Service Science},
+  pages={87--97},
+  year={2020},
+  organization={Springer},
+  url={https://lib.dr.iastate.edu/cgi/viewcontent.cgi?article=1187&context=imse_conf}
+}
+
+@article{tipping1999relevance,
+  title={The relevance vector machine},
+  author={Tipping, Michael},
+  journal={Advances in neural information processing systems},
+  volume={12},
+  year={1999},
+  url={https://proceedings.neurips.cc/paper/1999/file/f3144cefe89a60d6a1afaf7859c5076b-Paper.pdf}
+}
+
+@misc{scikitlearn2022ames,
+  title={The Ames housing dataset},
+  author={Scikit-Learn developers},
+  year={2022},
+  url={https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_ames_housing.html}
+}
+
+@misc{scikitlego2019fairness,
+  title={Fairness},
+  author={Vincent Warmerdam, Matthijs Brouns, and Scikit-Lego contributors},
```

Comment:
I'm not sure this is how BibTeX expects the author list to be formatted. See some of the other entries